.jenkins/nightly.groovy

```diff
@@ -31,7 +30,6 @@ pipeline {
                         -DCMAKE_CXX_EXTENSIONS=OFF \
                         -DKokkosKernels_ENABLE_TESTS=ON \
                         -DKokkosKernels_ENABLE_EXAMPLES=ON \
-                        -DKokkos_ENABLE_HIP=ON \
```

Comment:
@lucbv I am not sure this was there in the first place.

Reply:
I am not sure what you mean by that, isn't what triggers `#cmakedefine KOKKOS_ENABLE_HIP` at configure time?

Reply:
But happens on the Kokkos side

Reply:
Good point x)